WcaOnRails/app/webpacker/components/Panel/Leader/GroupsManager.jsx

```diff
@@ -98,6 +102,40 @@ function GroupTable({ groupId }) {
           ))}
         </Table.Body>
       </Table>
+      <Modal
+        size="fullscreen"
+        onClose={() => {
+          setModelOpen(false);
+        }}
+        open={modalOpen}
+      >
+        <Modal.Content>
+          <Form>
+            <Form.Field
+              label="New Member"
+              control={WcaSearch}
+              value={newMemberUser}
+              onChange={(_, { value }) => setNewMemberUser(value)}
```

Comment:
You can use our `useInputState` hook instead and pass it as `onChange={setNewMemberUser}`

Reply:
Done